api/src/main/java/gov/va/vro/api/resources/MasResource.java

```diff
@@ -24,9 +24,14 @@
 
 import javax.validation.Valid;
 
-@RequestMapping(value = "/v1", produces = "application/json")
-@SecurityRequirement(name = "X-API-Key")
-@SecurityScheme(name = "X-API-Key", type = SecuritySchemeType.APIKEY, in = SecuritySchemeIn.HEADER)
+@RequestMapping(value = "/v2", produces = "application/json")
+@SecurityRequirement(name = "Bearer Authentication")
+@SecurityScheme(
+    name = "Bearer Authentication",
+    type = SecuritySchemeType.HTTP,
+    bearerFormat = "JWT",
+    scheme = "bearer",
+    in = SecuritySchemeIn.HEADER)
 @Timed
```

Comment:
Can you add a comment here explaining how we can obtain the metrics?

Also, since the calls return asynchronously, does this really do what we want?

Reply:
Even though the response will be immediate, the metrics will involve the call to LH (IAM) to verify the token. The metrics on the asynchronous steps after the response can be tracked using the audit information